casepro/backend/junebug.py

```diff
@@ -109,9 +109,25 @@ def __init__(self, json_data):
         if remote_language is not None:
             self.language, _, _ = remote_language.partition('_')
         self.name = json_data.get('details').get('name', None)
-        self.addresses = json_data.get('details').get('addresses')
         self.fields = {}
         self.groups = {}
+        addresses = json_data.get('details').get('addresses')
+        self.urns = []
+        for scheme, address in addresses.items():
+            scheme_addresses = []
+            for urn, details in address.items():
+                if 'optedout' in details and details['optedout'] is True:
+                    # Skip opted out URNs
+                    continue
+                if 'default' in details and details['default'] is True:
+                    # If a default is set for the scheme then only store the default
+                    scheme_addresses = [urn]
+                    break
+                scheme_addresses.append(urn)
+            for value in scheme_addresses:
+                if scheme == "msisdn":
```

Comment:
Junebug doesn't know anything about address schemes. If the address doesn't exist in the identity store, then when we create it we use the value in the settings file `IDENTITY_ADDRESS_TYPE` (which is currently set to default to `msisdn`, we should probably change it to default to `tel`, but that might mean changing a bunch of tests.)

So it is configurable (at least when casepro is the one creating the identity, and we don't have any other services that create identities that I know of)